frontend/packages/data-portal/app/components/Dataset/SampleAndExperimentConditionsTable/components/CellStrainInfoLink/CellStrainInfoLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from 'app/components/Link'
+import { OBO, WORMBASE } from 'app/constants/datasetInfoLinks'
+
+const WORMBASE_PATTERN = /WBStrain[0-9]{8}/
+
+export function CellStrainInfoLink({
+  value,
+  id,
+}: {
+  value?: string | null
+  id?: string | null
+}) {
+  if (!value) {
+    return <span>--</span>
+  }
+
+  if (id) {
+    const isWormbase = WORMBASE_PATTERN.test(id)
+    const isOBO = !isWormbase
```

Comment:
I did this so the logic below will match the logic in InfoLink.tsx.  It felt weird to have OBO be the positive case in one and the negative case in the other.  Open to feedback here.